test/gpu/native/studies/chop/chplGPU.skipif

```diff
@@ -31,4 +31,12 @@ if ! git clone ${CHOP_URL} --branch=${CHOP_BRANCH} --depth=1 2>gitClone.out; the
   exit
 fi
 
+# Apply the GPU diagnostics patch, skipif that fails.
```

Comment:
I think I'd try and write this more generically to just apply every patch in the `patches` directory. That way the next time something comes up you can just add a new patch there without having to rewrite the `.skipif` file. Also when it comes time to remove the patch you can just `git rm` it without updating this code.

I haven't tried it but I'm guessing this would be something like this:

`for p in  ../patches/*patch; do git -C ChOp apply $p; done`